package-system/googlebenchmark/benchmark-1.7.0.patch

```diff
@@ -0,0 +1,250 @@
+diff -N -a -u -r benchmark-1.7.0.orig/src/benchmark.cc benchmark-1.7.0/src/benchmark.cc
```

Comment:
This patch shouldn't be needed for building for the current set of O3DE support platforms.